sample-kotlin-model/src/main/kotlin/com/vimeo/dummy/sample_kotlin/KotlinSamples.kt

```diff
@@ -9,17 +9,10 @@ import com.vimeo.stag.UseStag
  * Created by restainoa on 5/8/17.
  */
 @UseStag
-class KotlinSamples {
-    var stringField: String? = null
-
-    var nonNullStringField: String = "default"
-
-    var intField: Int? = null
-
-    var longField: Long = 1
-
-    @SerializedName("boolean_field")
-    var booleanField: Boolean? = null
-
-    var testField: Any? = null
-}
\ No newline at end of file
+data class KotlinSamples(
+        var stringField: String? = null,
+        var nonNullStringField: String = "default",
+        var intField: Int? = null,
+        var longField: Long = 1,
+        @SerializedName("boolean_field") var booleanField: Boolean? = null
```

Comment:
In this data class, other fields are not annotated. Why does only this field require annotation?

Reply:
annotating fields is optional, however, I will add the annotation to the other fields since it is inconsistent here.